examples/rbd/groupsnapshot.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
+kind: VolumeGroupSnapshot
+metadata:
+  name: new-groupsnapshot-demo-2
```

Comment:
```suggestion
  name: rbd-groupsnapshot
```
LGTM,
Just  a tiny nit
Let's keep naming conventions the same.

Reply:
Sure, definitely nicer.